lib/src/containerenv.rs

```diff
@@ -48,6 +48,7 @@ pub(crate) fn get_container_execution_info(rootfs: &Dir) -> Result<ContainerExec
             "id" => r.id = v.to_string(),
             "image" => r.image = v.to_string(),
             "imageid" => r.imageid = v.to_string(),
+            "rootless" => r.rootless = Some(v.to_string()),
```

Comment:
Argh...I swear I'd tested this before.  Hmmm...

Reply:
I'm just relieved there is a simple explanation instead of some crazy weird bug!